src/common.css

```diff
@@ -156,6 +157,8 @@
 }
 
 .global-close {
+    position: relative;
+    z-index: 3;
```

Comment:
Why did this become necessary?

Reply:
because the background is a clickable area once the account info page is open. The `.global-close` would be unclickable without it in that scenario.